src/main/java/com/online/store/controllers/product/controllers/AddProductController.java

```diff
@@ -0,0 +1,32 @@
+package com.online.store.controllers.product.controllers;
+
+import com.online.store.lib.Injector;
+import com.online.store.model.Product;
+import com.online.store.service.ProductService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class AddProductController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("com.online.store");
+    private final ProductService productService =
+            (ProductService) injector.getInstance(ProductService.class);
```

Comment:
```suggestion
    private ProductService productService =
            (ProductService) injector.getInstance(ProductService.class);
```

Reply:
Let's remove these modifiers for all of the service objects

Reply:
you can leave them